src/ui/common/src/components/pages/workflows/components/MetricItem.tsx

```diff
@@ -98,20 +98,24 @@ const MetricItem: React.FC<MetricItemProps> = ({ metrics }) => {
     setExpanded(!expanded);
   };
 
+  // 0.875rem is the size of the ShowMore.
+  // Add this additional padding if we don't have more than 1 metric to keep the rows equal size.
   return (
     <Box>
       {metrics.length > 0 ? (
-        <>
+        <Box sx={metrics.length === 1 && { padding: '0.875rem 0 0.875rem 0' }}>
```

Comment:
Where did you get this number from?

We haven't used rem anywhere in our app so far, so this is a little strange.

Reply:
Ah, I see this in the inspector now.

Could you actually make a const variable for this? and then you can use backticks: 
```
`${showMoreFontSize} 0 ${showMoreFontSize} 0`
```